gapic-common/lib/gapic/rest/client_stub.rb

```diff
@@ -137,35 +145,33 @@ def make_http_request verb, uri:, body:, params:, options:, is_server_streaming:
         next_timeout = get_timeout deadline
 
         begin
-          base_make_http_request(verb,
-                                 uri: uri,
-                                 body: body,
-                                 params: params,
-                                 metadata: options.metadata,
-                                 timeout: next_timeout,
-                                 is_server_streaming: is_server_streaming,
-                                 &block)
+          ClientStub.base_make_http_request(connection: @connection, verb: verb, uri: uri, body: body,
```

Comment:
Honestly I think having one parameter per line was more readable. Are we running afoul of Rubocop method line limits?

Reply:
yes